src/main/java/com/bina/varsim/types/stats/RatioRecord.java

```diff
@@ -1,17 +1,26 @@
 package com.bina.varsim.types.stats;
 
+import com.fasterxml.jackson.annotation.JsonProperty;
+
 /**
  * Stores everything required to compute precision and recall
  */
 public class RatioRecord {
+    @JsonProperty(value = "_TP")
     private int _TP = 0; // True Positive
```

Comment:
Can you please fix the naming of variables to not start with _ and make the names more like `truePositives` and so on?